en/appbuilder/change-log.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Indigo Design App Builder - Change Log 
+_description: Indigo Design App Builder enables design and development teams to use various interactions when creating connections between screens and components in order to cover real application scenarios.
+_keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch UI kits, Interactions
+---
+# Indigo Design App Builder - Change Log 
+
+> [!NOTE]
+><b>Latest update was made on April 20th, 2020.
+
+## Official Release
+
+### General
+* Cloud integration and team collaboration - Accessing the App Builder can be done through the App tab in Indigo.Design cloud, where users will find their active application projects. From there, users are enabled to start a new app or to preview, rename, archive or duplicate an existing one
+* Using Sketch file as a starting point for a new application in the App Builder - Now, users are enabled to start a new application from an existing Sketch file by uploading it in the App Builder. 
+
+### UI components
+* Dialog
+* Combo
+* Tabs layout
+* Expansion Panel
+* Date Time Editor (as an Input type)
+* Button Group
+* Text Area
+* Hyperlink
+
+### Interactions
+* Open/Close - The "Open/Close" interaction lets users create interactions using overlay components, such as Dialog.
+* Show/Hide - "Show/Hide" is an interaction that can be applied to a component to show and hide another component visibility in the same view.
+
+### Using Data in the App Builder
+* Rest API -  enables users to upload a JSON file or provide a URL with the Data Source. The added data source can be used also in the repeaters to connect components to actual data source fields.
```

Comment:
REST API means you can access a REST API endpoint. Uploading a JSON file is not a REST API. You should reword it to `Data Sources - ...`